src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/custom.py

```diff
@@ -42,6 +43,205 @@ def get_server_location(server_name, resource_group_name):
 
 
 ###############################################
+#                sql                          #
+###############################################
+
+
+def capabilities_get(  # pylint: disable=too-many-arguments
+        client,
+        location_id,
+        status=None,
+        edition=None,
+        service_objective=None,
+        depth=None):
```

Comment:
"Depth" is a pretty abstract way of controlling the granularity of information here.  Perhaps you could accept a space-separated list of properties to include/exclude in the output?